functions/donation/thank-you.ts

```diff
@@ -32,27 +32,79 @@ const hasThankYouEnvVars = (env: unknown): env is ThankYouEnvVars => {
-        "Your generosity offers vital support in expanding our efforts to build an independent and free online publication on global development. Your donation will support the expansion of the online publication in close collaboration with our research colleagues at the University of Oxford and around the world.",
-        "Given your interest in our work, we hope you will continue to follow our progress via our newsletters – if you have not done so, we’d like to invite you to join: https://ourworldindata.org/subscribe.",
-        `Reader donations are essential to our work, providing us with the stability and independence we need, so we can focus on research and the development of our site. ${
-            data.showOnList
-                ? "In recognition of your support we will be delighted to include your name as part of our List of Supporters: https://ourworldindata.org/supporters. We will add your name the next time we update the list and the sum of your donation will not be disclosed."
-                : ""
-        }`,
-        "Thank you again for your support for Our World in Data, we look forward to taking the project to the next level and we hope that you will remain interested in our work.",
-        "Kind regards,\nThe Our World in Data Team",
+            "We really appreciate your ongoing support! You’ll receive a receipt each month after your payment is processed. If you’d like to cancel your recurring donation at any point, just email us at donate@ourworldindata.org and we’ll take care of that for you.",
+        data.showOnList &&
+            "In recognition of your support, we will be delighted to include your name as part of our List of Supporters: https://ourworldindata.org/funding. We will add your name the next time we update the list, which we do every few months. The amount of your donation will not be disclosed.",
+        "Stay connected with our work: Follow us on social media or sign up for one of our newsletters here: https://ourworldindata.org/#subscribe (As a valued donor, we may also share an occasional donor-only update with you, from which you may unsubscribe at any time.)",
+        `Are you a UK taxpayer? If so, you can increase the value of your donation to us by 25% — at no extra cost to you — through the UK government’s Gift Aid scheme. Simply enter your name and address and tick a box on this form to confirm your UK taxpayer status, and we’ll take care of the rest:\n\n https://docs.google.com/forms/d/e/1FAIpQLSe7Mgm70-3UiRfh_aEJCusWCabdBFPN7hXoXMnby_6aAJsJVg/viewform?usp=pp_url&entry.2044643286=gcdl_${data.customerId}.\n\nLet us know if you have any questions: donate@ourworldindata.org`,
```

Comment:
May as well get rid of the inline `\n\n`'s here and store them as separate strings? 🚲 🖌️

Reply:
Good catch, that's a remnant from the time the whole part was conditional.

Reply:
Changed.